### PR TITLE
Work and about section complete

### PR DESCRIPTION
**Pull request by roshanasingh4**
Hello,

I am happy to inform you that I have completed the "About" and "Works" sections of the project in accordance with the provided instructions. Throughout the process, I have adhered to the best practices to ensure that the content is organized, concise, informative, and presented in a professional manner.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,5 +35,188 @@ <h2 class="intro">I’m a software developer</h2>
...
+            </div>
+
+            <div class="work_list">
+                <div class="state_capture">
+                </div>
+                <div class="particulars>
+                    <h2>Multi-Post Stories Gain+Glory</h2>
+                    <ul class="tech-stack">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
```
Comment (Olamarx) [inline index.html:91]: - [ ] This class is not closed and as a result the content is not as others.

## the third card
image

File: index.html [hunk 2]
```diff
@@ -35,5 +35,188 @@ <h2 class="intro">I’m a software developer</h2>
...
+       
+      
+    </section>
+    <section class="about_data">
+        <div class="about_info">
+            <h2>About me</h2>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+            </p>
+            <button >Get my resume</button>
```
Comment (Olamarx) [inline index.html:160]: - [ ] Kindly ket this button follow the interaction: Active, Hover, and Pressed.

## Figma interaction
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @roshanasingh4,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️ 
- Readme file professional ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
[OPTIONAL] I would implore you to add the Future features header to the pointed space in your readme file. You must have forgotten. Nice work 👍🏼
image


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @roshanasingh4 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.
-  [ ] Please always ensure to always complete the project requirements before submitting for Code Review. This disturbs the review process. Once you have submitted the project for review, you must avoid applying other changes ☺️.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**